tests/bmp/test_bmp_statedb.py

```diff
@@ -13,6 +15,19 @@
 ]
 
 
+def get_mgmt_ip(duthost):
+    config_facts = duthost.get_running_config_facts()
+    mgmt_interfaces = config_facts.get("MGMT_INTERFACE", {})
+    mgmt_ip = None
+
+    for mgmt_interface, ip_configs in mgmt_interfaces.items():
+        for ip_addr_with_prefix in ip_configs.keys():
+            ip_addr = ip_addr_with_prefix.split("/")[0]
+            if valid_ipv4(ip_addr):
```

Comment:
>valid_ipv4 [](http://example.com/codeflow?start=15&length=10)

There are some special DUTs:
1. M0 do not have mgmt port at all, expecting to query loop IP address
2. Some device only has ipv6 mgmt address

Suggest loop all the access if available:
1. mgmt port ipv4
2. mgmt port ipv6
3. loopback ipv4
4. loopback ipv6